CHANGELOG.md

```diff
@@ -1,5 +1,53 @@
 For 1.x changes, go [here](https://github.com/twilio/twilio-video.js/blob/support-1.x/CHANGELOG.md).
 
+2.0.0-beta8 (in progress)
+=========================
+
+New Features
+------------
+
+- You can now use the [Network Quality](https://www.twilio.com/docs/video/using-network-quality-api)
+  API to control the verbosity of the network quality statistics that are reported
+  for the LocalParticipant and RemoteParticipants in a Group Room. A Participant will
+  now have an additional property `networkQualityStats` which contains the network
+  quality statistics used to calculate the `networkQualityLevel`. (JSDK-2255)
+ 
+  ```js
+  const { connect } = require('twilio-video');
+  const room = await connect(token, {
+    networkQuality: {
+      local: 1, // Verbosity level for LocalParticipant [1 - 3]
```

Comment:
Should we have these as "localVerbosityLevel" and "remoteVerbosityLevel"?

Reply:
@aymenn ,

The names are formalized in the [IDL schema](https://code.hq.twilio.com/client/video-sdk-api/blob/master/idl/networkqualityconfiguration.md).